### DISCUSSION TITLE
Should  OIDCScope accept scopes without openid?

### DISCUSSION THREAD

**Discussion by iplparm** [OpenIDC/mod_auth_openidc#1066] [General]
Hi,
I'm able to set OIDCSCope to any value and it's in line with the docs:
https://github.com/OpenIDC/mod_auth_openidc/blob/44ede94193c8c8fa32204911f4a689fa8e640685/auth_openidc.conf#L140
If nothing is set "bare minimum" scope "openid" will be added by the module.
However, if you set OIDCScope to any value(s) and won't include "openid" scope - module accepts such configuration but generates runtime errors on every response validation: https://github.com/OpenIDC/mod_auth_openidc/blob/44ede94193c8c8fa32204911f4a689fa8e640685/src/proto.c#L2737

Should the module fall back to OAuth if "openid" scope is not set deliberately or is it purposed to handle only OIDC flows and I should always add "openid"?

Best regards,
Arek Wojtczak

**Comment by zandbelt** [comment]
you'll need to add "openid" yourself if you modify the default

**Reply by iplparm** [reply]
Thank you.
